uast.h

```diff
@@ -1,31 +1,58 @@
 #ifndef _NODE_API
 #define _NODE_API
 
-#include <string>
-#include <vector>
+#ifdef __cplusplus
```

Comment:
Since this is a C library, this should probably not be here. A C++ user may want to define it themself if they need to.